src/typegoose.ts

```diff
@@ -1,14 +1,8 @@
 /* imports */
 import * as mongoose from 'mongoose';
 import 'reflect-metadata';
-import * as semver from 'semver';
 import { deprecate, format, isNullOrUndefined } from 'util';
 
-/* istanbul ignore next */
-if (semver.lt(mongoose.version, '5.7.7')) {
```

Comment:
why remove this?

Reply:
This is an error, there is no reason to remove this.

Reply:
Reverted in an commit